src/mlpack/core/util/version.hpp

```diff
@@ -18,7 +18,7 @@
 // with higher number than the most recent release.
 #define MLPACK_VERSION_MAJOR 3
 #define MLPACK_VERSION_MINOR 0
-#define MLPACK_VERSION_PATCH 3
+#define MLPACK_VERSION_PATCH 5
```

Comment:
Again a minor query, shouldn't this be `4`?

Reply:
The documentation says we will keep it one version newer than the latest stable version in the git repository.  Honestly I don't have too much of an opinion here so we could change it if we wanted.

Reply:
I think this is just fine.